lib/bugcrowd_templates.rb

```diff
@@ -40,15 +40,52 @@ def get(
       file_name: file_name
     ).template_file(type)
 
-    template_data(template_path)
+    template_data(file_path: template_path, category: category, subcategory: subcategory, item: item)
   end
   # rubocop:enable Metrics/ParameterLists
 
   def current_directory
     DATA_DIR
   end
 
-  def template_data(file_path)
-    File.open(file_path).read if File.exist?(file_path)
+  def template_data(file_path:, category:, subcategory:, item:)
+    if File.exist?(file_path)
+      File.open(file_path).read
+    else
+      find_template(file_path, category, subcategory, item)
+    end
+  end
+
+  def find_template(file_path, category, subcategory, item)
+    return File.open(file_path).read if File.exist?(file_path)
+
+    file_path = get_pathname(file_path)
+    presented_template(file_path, category, subcategory, item)
+  end
+
+  def get_pathname(file_path)
+    Pathname.new(file_path)
+  end
+
+  def get_parent_path(file_path)
+    File.dirname(file_path.parent)
+  end
+
+  def get_file_basename(file_path)
+    File.basename(file_path)
+  end
```

Comment:
using `file_name` i will get only `template` or `guidance`. we need file_name  with extension `template.md`, so that we can combine the  file_path with basename.

Reply:
fixed